core-bundle/src/Resources/contao/templates/_new/content_element/table.html.twig

```diff
@@ -0,0 +1,79 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{# todo: remove once the table sorting script was adjusted or inlined #}
+{% set attributes = attrs().addClass('ce_table') %}
+
+{% block content %}
+    {% set table_attributes = attrs(table_attributes|default)
+        .set('id', 'table_' ~ data.id, sorting)
+        .addClass('sortable', sorting)
+        .set('data-sort-default', sorting.column|default ~ '|' ~ sorting.order|default, sorting)
+    %}
+    <table{% block table_attributes %}{{ table_attributes }}{% endblock %}>
+        {# Caption #}
+        {% if caption %}
+            {% block caption %}
+                <caption>{% block caption_inner %}{{ caption|insert_tag_raw }}{% endblock %}</caption>
+            {% endblock %}
+        {% endif %}
+
+        {# Header #}
+        {% if header %}
+            {% block header %}
+                <thead>
+                {% block header_inner %}
+                    <tr>
+                        {% for col in header %}
```

Comment:
Maybe rename `col` to `cell` everywhere in this template?

Reply:
see ea7409be00c212879e48208b6dd86f63f8b89ee2